<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Image_Detection</class>
 <widget class="QWidget" name="Image_Detection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Detection</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Courier 10 Pitch&quot;;
color: rgb(52, 101, 164);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>  Image Objects Detection</string>
   </property>
  </widget>
  <widget class="QLabel" name="new_image">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>270</y>
     <width>450</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(52, 101, 164);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="original_image">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>450</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(52, 101, 164);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="convert">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>510</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(173, 127, 168);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="path">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(209, 48, 48);
color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Add relative Path of the considered image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
